.gemini/commands/find-docs.toml

```diff
@@ -0,0 +1,27 @@
+description = "Find relevant documentation and output GitHub URLs."
+
+prompt = """
+## Mission: Find Relevant Documentation
+
+Your task is to find documentation files relevant to the user's question (`<QUESTION>`) within the current git repository and provide a list of GitHub URLs to view them.
```

Comment:
You could utilize `{{args}}` to positionally represent the users question if you'd like